x-pack/beatless/magefile.go

```diff
@@ -0,0 +1,83 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build mage
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+
+	"github.com/elastic/beats/dev-tools/mage"
+)
+
+func init() {
+	mage.SetBuildVariableSources(mage.DefaultBeatBuildVariableSources)
```

Comment:
Is this line required? (Since it's in our repo hopefully it just works without setting this.)